<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalibrator</class>
 <widget class="QWidget" name="CameraCalibrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraCalibrator</string>
  </property>
  <widget class="QPushButton" name="Exit_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="ModeInfo_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose a mode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SingleCalib_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Single Camera Calibration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StereoCalib_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Stereo Camera Calibration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CalibErrorTesting_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration Error Testing</string>
     </property>
    </widget>
    <widget class="QLabel" name="Logo_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>241</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource.qrc">:/res/logo.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select a camera source</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>104</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>V4L2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RTSP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>GIGE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
